src/python/foglamp_daemon.py

```diff
@@ -21,9 +23,7 @@ def do_something(logf):
     logger.addHandler(fh)
     logger.setLevel(logging.DEBUG)
 
-    # set DB config
-    Configurator().initialize_dbconfig()
-    CoAPServer.start()
+    start.start()
```

Comment:
`starter.start()` is better than `start.start()` or may be runserver.start() after renaming starter.py to  runserver.py